src/core/public/chrome/ui/header/header_breadcrumbs.scss

```diff
@@ -13,43 +13,85 @@ $breadcrumbPolygon: polygon(
   0% 100%
 );
 
-.osdHeaderBreadcrumbs {
-  /*
-  filter defines a custom filter effect by grouping atomic filter primitives!
-  here we are using Gaussian filter with stdDeviation for applying
-  border-radius on clipped background.
-  */
-
-  filter: url("../../public/assets/round_filter.svg#round");
+/* remove background color on autofocus for euiBreadcrumbs in Popover */
+.euiBreadcrumbs__inPopover {
+  .euiLink.euiLink--text:focus {
+    background: none;
+  }
+}
```

Comment:
So this behavior change is intended for both light and dark mode?

Reply:
to be consistent!

Reply:
will confirm with ux team